internal/otgutils/print.go

```diff
@@ -144,3 +144,75 @@ func LogLACPMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
 	out.WriteString("\n\n")
 	t.Log(out.String())
 }
+
+// LogLLDPMetrics is displaying otg lldp stats.
+func LogLLDPMetrics(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
+	t.Helper()
+	var out strings.Builder
+	out.WriteString("\nOTG LLDP Metrics\n")
+	fmt.Fprintln(&out, strings.Repeat("-", 120))
+	out.WriteString("\n")
+	fmt.Fprintf(&out,
+		"%-15s%-15s%-15s%-18s%-20s%-18s%-18s\n",
+		"Name",
+		"Frames Tx",
+		"Frames Rx",
+		"Frames Error Rx",
+		"Frames Discard",
+		"Tlvs Discard",
+		"Tlvs Unknown")
+
+	for _, lldp := range c.Lldp().Items() {
+		lldpMetric := gnmi.Get(t, otg, gnmi.OTG().LldpInterface(lldp.Name()).Counters().State())
+		framesTx := lldpMetric.GetFrameOut()
+		framesRx := lldpMetric.GetFrameIn()
+		framesErrorRx := lldpMetric.GetFrameErrorIn()
+		framesDiscard := lldpMetric.GetFrameDiscard()
+		tlvsDiscard := lldpMetric.GetTlvDiscard()
+		tlvsUnknown := lldpMetric.GetTlvUnknown()
+		out.WriteString(fmt.Sprintf(
+			"%-15v%-15v%-15v%-18v%-20v%-18v%-18v\n",
+			lldp.Name(), framesTx, framesRx, framesErrorRx, framesDiscard, tlvsDiscard, tlvsUnknown,
+		))
+	}
+	fmt.Fprintln(&out, strings.Repeat("-", 120))
+	out.WriteString("\n\n")
+	t.Log(out.String())
+}
+
+// LogLLDPNeighborStates is displaying otg lldp neighbor states.
+func LogLLDPNeighborStates(t testing.TB, otg *otg.OTG, c gosnappi.Config) {
+	t.Helper()
+	var out strings.Builder
+	out.WriteString("\nOTG LLDP Neighbor States\n")
+	fmt.Fprintln(&out, strings.Repeat("-", 120))
+	out.WriteString("\n")
+	fmt.Fprintf(&out,
+		"%-15s%-18s%-18s%-18s%-20s%-20s\n",
+		"LLDP Name",
+		"System Name",
+		"Port Id",
+		"Port Id Type",
+		"Chassis Id",
+		"Chassis Id Type")
+
+	for _, lldp := range c.Lldp().Items() {
+		lldpNeighborStates := gnmi.GetAll(t, otg, gnmi.OTG().LldpInterface(lldp.Name()).LldpNeighborDatabase().LldpNeighborAny().State())
+		for _, lldpNeighborState := range lldpNeighborStates {
+			systemName := lldpNeighborState.GetSystemName()
+			portId := lldpNeighborState.GetPortId()
+			portIdType := lldpNeighborState.GetPortIdType()
+			chassisId := lldpNeighborState.GetChassisId()
+			chassisIdType := lldpNeighborState.GetChassisIdType()
```

Comment:
Can you change "Id" to "ID" in these vars? I see these flagged in static analysis, not sure if you see them in your view.

We want the case to match on these, similar to initialisms.

https://google.github.io/styleguide/go/decisions#initialisms

(I know you you're matching the field names, unfortunately they're autogenerated so we can't enforce the style guide)